TA 97932 (#16515)
cockroachdb/docs: v22.2/security-reference/scram-authentication.md

{% include_cached {{page.version.version}}/scram-authentication-recommendations.md %}

> In clusters running v22.2.0 to v22.2.6, the default hashing cost was high for certain clients that are running on a limited amount of CPU, which may cause higher than normal connection latency. See the [Technical Advisory](../../advisories/a97932.html) for mitigation steps. 

Reviewer: ```suggestion
In clusters running v22.2.0 to v22.2.6, the default hashing cost was too high for certain clients running on a limited amount of CPU, causing connection latency. See the [Technical Advisory](../../advisories/a97932.html) for mitigation steps. 
```
